node_modules/gh-events/lib/restmodel.js

```diff
@@ -56,21 +56,6 @@
 /**
  * @RESTModel EventOrganiserUpdate
  *
- * @Required []
- * @Property  {EventOrganiserUpdateOther}   organiserOther      Used when the organiser is not a recognised user
- * @Property  {EventOrganiserUpdateUser}    organiserUsers      Used when the organiser(s) is a recognised user
- */
-
-/**
- * @RESTModel EventOrganiserUpdateOther
- *
- * @Required  []
- * @Property  {boolean}                     {organiserUser}     Whether the named user should be an organiser
- */
-
-/**
- * @RESTModel EventOrganiserUpdateUser
- *
  * @Required  []
- * @Property  {boolean}                     {organiserOther}    Whether the named user should be an organiser
+ * @Property  {boolean}                     {organiserUser}     Whether the user should be an organiser
```

Comment:
Because what was outlined in those docs isn't actually possible through REST. I think you'd have to submit a JSON blob and parse it on the backend in order to achieve that.